src/streamingCalls/streaming.ts

```diff
@@ -199,6 +215,9 @@ export class StreamProxy extends duplexify implements GRPCCallResult {
     }
 
     const stream = apiCall(argument, this._callback) as CancellableStream;
+    if (stream instanceof Promise) {
```

Comment:
This is another place called `apiCall`, but it the code after line 217 never been called.

Reply:
Still makes sense to attach the `.catch` handler here as well. So maybe just do

```
function ensureStream(stream: CancelableStream | Promise) {
  if (typeof stream === "Promise") {
    const replace = new PassThrough();
    stream.catch(err => {
      replace.emit('error', new GoogleError(err));
    });
    return replace;
  }
  return stream;
}
```

and then in both places,
```
stream = ensureStream(stream);
...
return stream;
```